capplets/about-me/mate-about-me.c

```diff
@@ -410,12 +410,10 @@ about_me_fingerprint_button_clicked_cb (GtkWidget *button, MateAboutMe *me)
 #if HAVE_ACCOUNTSSERVICE
 static void on_user_is_loaded_changed (ActUser *user, GParamSpec *pspec, MateAboutMe* me)
 {
-        if (act_user_is_loaded (user)) {
-		about_me_load_photo (me);
-		g_signal_handlers_disconnect_by_func (G_OBJECT (user),
-				G_CALLBACK (on_user_is_loaded_changed),
-				me);
+	if (!act_user_is_loaded (user)) {
+		return;
 	}
+	about_me_load_photo (me);
```

Comment:
Anyway changes to `on_user_is_loaded_changed` is unneeded, no? It does same, except removed `g_signal_handlers_disconnect_by_func`...